x-pack/plugin/profiling/src/test/java/org/elasticsearch/xpack/profiling/GetStackTracesRequestTests.java

```diff
@@ -28,47 +24,6 @@
 import static java.util.Collections.emptyList;
 
 public class GetStackTracesRequestTests extends ESTestCase {
-    public void testSerialization() throws IOException {
```

Comment:
Instead of removing this test, can you rename/replace it with checks for the setter/getter functions, e.g. a set value is expected to be returned by the get function (we had an issue already where a getter returned the wrong value).
Also testing the getters with `null` values should be done.

But maybe we cover these code paths already with some other tests.
A test coverage report would strongly help improving tests. Do we have it somewhere in the CI reports already?

Reply:
> I'm not aware that such reports exist.

Too bad. I'd consider a test coverage report as a very helpful tool to improve code quality. It also should be a low-hanging fruit for CI / automation.